packages/builder/src/builder.js

```diff
@@ -235,6 +235,8 @@ export default class Builder {
     // Plugins
     this.plugins = Array.from(this.normalizePlugins())
 
+    this.resolvePluginsMode()
```

Comment:
We can move the call to https://github.com/nuxt/nuxt.js/blob/17259d5f617214cac140e51809958bdcc41322ac/packages/builder/src/builder.js#L275